exploration/pattern-exterior-whitespace.md

```diff
@@ -53,6 +57,101 @@ Examples include:
 - indenting text
 - manually aligning multi-line text that is rendered with a monospace font
 
+### Other whitespace characters
+
+Note that there are many whitespace characters in Unicode, such as U+3000,
+U+200A, U+200B, U+2025, etc. which are not included in the `s` production.
+These whitespace characters are part of the _pattern_ and are not subject to being
+trimmed in syntaxes that support unquoted patterns.
+Because these whitespace characters look like spaces (or nothing)
+and can be followed by characters that would be pattern exterior whitespace,
+in some _messages_ users might be surprised by the lack of trimming.
+Note that this is true of any non-ASCII whitespace characters, however,
+when viewed as plain text.
+
+For example:
+>```
+> %match {$foo}
+> %{when *} \u200a  \nHello {$foo}
+>```
+>The `*`-keyed _pattern_ starts with a `HAIRSPACE` followed by two ASCII spaces and a newline.
+>The AScII space (U+0020) before the `\u200a` character is _pattern exterior whitespace_.
+>The `\u` escapes are not part of MFv2's syntax and are for visibility.
+>What this message actually looks like is:
+>```
+>%match {$foo}
+>%{when *}
+>Hello {$foo}
```

Comment:
Including the `\n` is mostly a distraction here, and it makes the preceding whitespace be handled by line-end trimming where that's applicable. Also, the "actually looks like" ought to include the spaces it's talking about.

```suggestion
> %match {$foo}
> %{when *} \u200a  Hello {$foo}
>```
>The `*`-keyed _pattern_ starts with a `HAIRSPACE` followed by two ASCII spaces.
>The ASCII space (U+0020) before the `\u200a` character is _pattern exterior whitespace_.
>The `\u` escapes are not part of MF2 syntax and are for visibility.
>What this message actually looks like is:
>```
>%match {$foo}
>%{when *}    Hello {$foo}
```